### ISSUE TITLE
does it work with angular 6 ?

### ISSUE THREAD

**Issue by jhonyourangel** [akveo/ng2-smart-table#780]
i get a few errors:
````
ERROR in node_modules/ng2-smart-table/lib/data-source/data-source.d.ts(1,10): error TS2305: Module '"/Users/aiu/Desktop/testTable/node_modules/rxjs/Subject"' has no exported member 'Subject'.
node_modules/ng2-smart-table/lib/data-source/data-source.d.ts(2,10): error TS2305: Module '"/Users/aiu/Desktop/testTable/node_modules/rxjs/Observable"' has no exported member 'Observable'.
node_modules/rxjs/Observable.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Observable'.
node_modules/rxjs/Subject.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Subject'.
````

**Comment by luhtonen** [comment]
Will this error go away if you add `rxjs-compat` package? I should add backward compatibility, but I don't know will it work with 3rd party packages.

Here's the link to blog [about migration to newer RxJS](https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md)

**Comment by jhonyourangel** [comment]
i haven't tried with smart-table yet, but with a small project that i've started last week `rxjs-compat` didn't help.
i will try this tomorrow morning.

**Comment by nitinht1988** [comment]
Getting the same error after upgrading to angular 6...Please help!!!

**Comment by jhonyourangel** [comment]
for everyone that has this problem with angular 6 just run ` npm install rxjs-compat` is should do the trick
it did for me

**Comment by luhtonen** [comment]
> Installing rxjs-compat is not a proper solution. It should work without that.
No, it is not, but it's a good workaround until proper fix will be implemented.

**Comment by allankz** [comment]
Hi, I solved the problem to work on Angular 6.
Find and replace ALL "rxjs/*" from all files (ie: "rxjs/Observable" or things like 'rxjs/add/operator')
It must be done also on ng2-completer module.

**Comment by giansalex** [comment]
This library need to update to Angular 6

**Comment by Tenmak** [comment]
Yes, and the akveo team said that this will only be community supported, so if nobody moves, this will never get updated. 

Better off switching to another angular table imho.

`Hi, I solved the problem to work on Angular 6.
Find and replace ALL "rxjs/*" to "rxjs" only. Do it for all files (ie: "rxjs/Observable" or things like 'rxjs/add/operator')
It must be done also on ng2-completer module.`

Great, do `npm install` and you can do it again.

**Comment by nnixaa** [comment]
Hey Guys, there is a PR already https://github.com/akveo/ng2-smart-table/pull/786 so we hopefully will be able to update it quite soon.

**Comment by nnixaa** [comment]
published to npm as 1.3.0
